_docs_integrate/attribute-introduction.md

```diff
@@ -21,3 +21,132 @@ required_by:
+
+#### Peer shared IdentityAttributes
+
+If an [IdentityAttribute]({% link _docs_integrate/data-model-overview.md %}#identityattribute) is shared by its `owner`, this not only leads to the creation of an own shared IdentityAttribute as a copy of the associated RepositoryAttribute for the `owner` of the IdentityAttribute, but also to the creation of a peer shared IdentityAttribute for the peer with whom the IdentityAttribute was shared. A peer shared IdentityAttribute is a [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute) for which the `shareInfo` property is set. The Address of the `owner` of the IdentityAttribute is contained within its `shareInfo.peer` property. The `shareInfo.sourceAttribute` property of a peer shared IdentityAttribute is always undefined, as the RepositoryAttribute used as the source is only stored locally for the `owner` of the IdentityAttribute. Furthermore, note that the `id` of a peer shared IdentityAttribute is always the same as the `id` of the associated own shared IdentityAttribute. To ensure the privacy of an Identity's data, the IdentityAttribute shared by its `owner` with a peer cannot be shared by that peer with a third party.
+
+## RelationshipAttributes
+
+A [RelationshipAttribute]({% link _docs_integrate/data-model-overview.md %}#relationshipattribute) is used to store data that is relevant in the context of a [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) between two Identities. Both Identities involved in the Relationship must agree to its creation.
+In the context of a single Relationship, each RelationshipAttribute has its unique `key` for identification.
```

Comment:
I discussed this with a co-worker a month ago. Apparently there should only be one RelationshipAttribute of a `key` per `owner` within a Relationship, which would also make sense. In fact, this has not yet been implemented or validated, respectively. Together with my co-worker, we decided to put this on the back burner for the time being and have noted it as a to-do instead. 

Let's both implement this feature together soon? The sentence here in the documentation could be left as it is, as this would be good practice when creating RelationshipAttributes. What is your intuition on whether only one  RelationshipAttribute of a certain `key` can be used within a Relationship or at most two, if the `owner` is different? Presumably we have to use the second option anyway because of the third party owned RelationshipAttributes, i.e. take into account differences in ownership?

Reply:
It's fine for me if you leave it here as it is and we take care of the implementation soon.